tests/test_nc_to_mmd.py

```diff
@@ -536,6 +537,26 @@ def test_publication_date(self):
         dt = datetime.datetime.strptime(value[0]['publication_date'], format)
         self.assertEqual(dt, datetime.datetime(2020, 11, 27, 0, 0))
 
+    def test_checksum(self):
+        tested = tempfile.mkstemp()[1]
+        fn = 'tests/data/reference_nc.nc'
+        nc2mmd = Nc_to_mmd(fn, check_only=True)
+        nc2mmd.to_mmd()
+        checksum = nc2mmd.metadata['storage_information']['checksum']
+        with open(tested, 'w') as tt:
+            tt.write('%s *%s'%(checksum, fn))
+        md5hasher = FileHash('md5')
+        self.assertTrue(md5hasher.verify_checksums(tested)[0].hashes_match)
+
+    def test_checksum_is_equal_as_stored(self):
+        # NOTE: if the netcdf file is changed, this test will fail
+        fn = 'tests/data/reference_nc.nc'
+        nc2mmd = Nc_to_mmd(fn, check_only=True)
+        nc2mmd.to_mmd()
+        checksum = nc2mmd.metadata['storage_information']['checksum']
+        with open('tests/data/hashes.md5', 'r') as f:
+            ll=f.readline()
+        self.assertEqual(ll.split(' ')[0], checksum)
 
```

Comment:
I dont understand the point of this test. Is it to detect if the reference file has changed?

Reply:
yes - partly out of interest... It is probably not strictly necessary. Should I remove it?

Reply:
I thin so

Reply:
ok